lib/evmone/baseline_instruction_table.cpp

```diff
@@ -34,6 +34,7 @@ constexpr auto legacy_cost_tables = []() noexcept {
     tables[EVMC_PRAGUE][OP_DATALOADN] = instr::undefined;
     tables[EVMC_PRAGUE][OP_DATASIZE] = instr::undefined;
     tables[EVMC_PRAGUE][OP_DATACOPY] = instr::undefined;
+    tables[EVMC_PRAGUE][OP_RETURNDATALOAD] = instr::undefined;
```

Comment:
It could potentially, but we don't have it in a separate EIP, just alongside the revamped calls, which also are batched together with the rest of EOF. In this way RETURNDATALOAD is sort of "attached" to EOF.

Supporting this in legacy would mean we need to do the entire dance with backwards compatibility (a previously undefined instruction), correct? Maybe it is a pragmatic choice to only have it in EOF. I'm not sure what is the usual practice in such cases.

Reply:
Revamped CALLS are activated in legacy, too. 

The same reasoning applies here: RETURNDATALOAD does not require immediates, and can be useful for legacy, too. And it's even in the same EIP so far.

Usually not much backwards compatibility dancing is happening when introducing new instructions. From EIP:
>  Backwards Compatibility
>
>No existing instructions are changed and so we do not think any backwards compatibility issues can occur.

Reply:
In the end it was decided to not activate revamped CALLS in legacy, so RETURNDATALOAD will go down the same path.